Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:10.16.0-alpine AS build
+WORKDIR /usr/src/app
+COPY ["package.json", "package-lock.json*", "./"]
+COPY . .
+RUN npm run build
+
+FROM node:10.16.0-alpine
+WORKDIR /usr/src/app
+ENV NODE_ENV=production
+COPY --from=build ./usr/src/app/proto ./proto
+COPY --from=build ./usr/src/app/dist ./dist
+COPY --from=build ./usr/src/app/package.json ./
+# Use the local node_modules because the package @rocket.chat/sdk drops the build due to an unknown problem with it
```

Comment:
show me the error, sounds weird to me...

Reply:
@rocket.chat/sdk this is the problematic package.
The command '/bin/sh -c npm install --silent' returned a non-zero code: 254.
The problem is unknown but hi team recommended this package.

Reply:
A good solution was not found so I guess yes...